tools/chocolateyinstall.ps1

```diff
@@ -4,22 +4,35 @@ $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
 Get-ChildItem -Path "$toolsDir\tailscale.cer" | Import-Certificate -CertStoreLocation Cert:\LocalMachine\TrustedPublisher
 
-$url = 'https://pkgs.tailscale.com/stable/tailscale-ipn-setup-1.34.1.exe'
-
-
 $packageArgs = @{
-  packageName    = $env:ChocolateyPackageName
-  unzipLocation  = $toolsDir
-  fileType       = 'exe'
-  url            = $url
-
-  softwareName   = 'Tailscale'
-
-  checksum       = '302b5ef05ad97d39c81624d77882b67adfef2e68da4c9e2a9ebbf5667c29a3b0'
-  checksumType   = 'sha256'
+  SoftwareName   = 'Tailscale'
+  PackageName    = $env:ChocolateyPackageName
+  UnzipLocation  = $toolsDir
+  FileType       = 'msi'
 
-  silentArgs     = '/S'
+  silentArgs     = '/quiet'
   validExitCodes = @(0)
 }
 
+if ($env:PROCESSOR_ARCHITECTURE -eq 'ARM64') {
+  # Windows on ARM
+  $packageArgs += @{
+    Url          = 'https://pkgs.tailscale.com/stable/tailscale-setup-1.34.2-arm64.msi'
```

Comment:
Based upon what I've been reading, this line should probably use `Url64Bit` instead of `Url`...

Reply:
I don't think so. Look up at the previous discussion, specifically [here](https://github.com/tailscale/tailscale-chocolatey/pull/60#discussion_r1061841832). On arm64, Chocolatey apparently tries to install the 32bit option. Either way, as the only option when running on arm64, it should install fine.

Reply:
The GUI in the ARM64 build is still 32-bit x86 so this is probably still technically correct as is anyway.